### PR TITLE
add miles text to radius select

### PR DESCRIPTION
**Pull request by cmertan**

### CODE REVIEW

**Review by anna-dorigo — COMMENTED** [review]
I think you just need to close the string, add a closing double quote: "1 Miles"

File: app/views/component/search/_geolocate.html.haml [hunk 1]
```diff
@@ -2,8 +2,7 @@
   = label_tag :distance, 'Distance', style: "font-size: 16px;"
   %hr
   = label_tag 'within', t('labels.distance_filter')
-  = select_tag :distance, options_for_select([1, 2, 10, 25], @selected_distance_filter), include_blank: 'All', class: "category-search-select-left-menu"
-
+  = select_tag :distance, options_for_select([["1 mile, 1], ["2 miles, 2], ["10 miles, 10], ["25 miles, 25]], @selected_distance_filter), include_blank: 'All', class: "category-search-select-left-menu"
```
Comment (anna-dorigo) [inline app/views/component/search/_geolocate.html.haml:5]: Something like: 
`= select_tag :distance, options_for_select([["1 mile", 1], ["2 miles", 2], ["10 miles", 10], ["25 miles", 25]], @selected_distance_filter), include_blank: 'All', class: "category-search-select-left-menu"`

Comment (cmertan) [reply app/views/component/search/_geolocate.html.haml]: fixed!

**Review by anna-dorigo — APPROVED** [review]
Tested locally and works fine👏🏽

Screen Shot 2021-02-11 at 11 09 26 AM

**Pull request merged**